UserDetails/index.js

```diff
@@ -256,8 +255,8 @@ export default class Plugin extends BasePlugin {
             }
 
             switch (props.type) {
-                case ActivityTypes.PLAYING: return <Gamepad {...props} />;
-                case ActivityTypes.LISTENING: return <Headphones {...props} />;
+                case "PLAYING": return <Gamepad {...props} />;

```

Comment:
The module actually still exists for this.
![image](https://user-images.githubusercontent.com/46447572/182435827-9a6fd2dc-4468-461a-81d6-4206c8af5264.png)

Reply:
Somehow i thought that is ActionTypes. my bad